tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -512,9 +513,14 @@ struct RTFFile {
 /// A zero UUID stored at the end of the TFile record
 struct RTFUUID {
    RUInt16BE fVersionClass{1};
-   unsigned char fUUID[16] = {0};
+   unsigned char fUUID[16];
 
-   RTFUUID() = default;
+   RTFUUID()
+   {
+      TUUID uuid;
+      char *buffer = reinterpret_cast<char *>(this);
+      uuid.FillBuffer(buffer);
```

Comment:
`sizeof(*this)` is different from `sizeof(TUUID)` because `TUUID` has transient members (its size is 20, but the on-disk size is 16)

Reply:
Fair enough.  Implicit was the question of whether we wanted the life object or only the ondisk representation.
Eitherway, I would strongly recommend adding the corresponding assert in case anything changes (unlikely at it may be).